### PR TITLE
CC-0049 A user can see titles on the editor page(TopBar Canva)

### PR DESCRIPTION
**Pull request by Fuchih**
clicking one of the court sizes in the blueprint

can see the text on the top left corner of the top bar change to the relevant court size

and can see the text on the canvas change to the relevant court size

image

image

### CODE REVIEW

File: src/components/BasketballCourt/index.tsx [hunk 1]
```diff
@@ -20,8 +23,12 @@ const BasketballCourt = () => {
       minHeight={400}
       justifyContent="center"
       alignItems="center"
+      flexDirection="column"
       margin="auto"
     >
+      <Heading as="h2" transform="translateY(37px)" fontWeight="medium" fontSize="20px">
+        {courtName}
+      </Heading>
```
Comment (tatoMa) [inline src/components/BasketballCourt/index.tsx:31]: This implementation is different from Beryl's. Please be aware of that. You both modified this part.

File: src/components/EditorSideBar/Blueprints.tsx [hunk 2]
```diff
@@ -1,11 +1,21 @@
...
+  const handleCourtName = (des: string, des1: string): void => {
+    dispatch(changeCourtName(`${des} ${des1}`));
+  };
 
-const Blueprints = () => {
   return (
     <Box pl="24px" pt="24px" height="100%" className="scrollbox">
       {courtList.map((court) => {
-        const { img } = court;
+        const { img, des, des1 } = court;
```
Comment (tatoMa) [inline src/components/EditorSideBar/Blueprints.tsx:18]: It is not your problem but des and des1 is not a proper name for variables. Maybe change them?

Comment (Fuchih) [reply src/components/EditorSideBar/Blueprints.tsx:18]: I do aware of this problem  I will create a new ticket to fix those naming issues at one time

**Review by tatoMa — APPROVED** [review]
LGTM

**Comment by jennawan** [comment]
image
A small error that in the courtList, the des1 of 150 m2 court should be "(15 m * **10** m)" instead of "(15 m * **28** m)", otherwise looks good to me!

**Review by GitHubEason — APPROVED** [review]
LGTM

**Comment by Fuchih** [comment]
> image A small error that in the courtList, the des1 of 150 m2 court should be "(15 m * **10** m)" instead of "(15 m * **28** m)", otherwise looks good to me!

not in this ticket scope. I will change it at ticket cc-0056 thanks!

**Pull request merged**